webapp/.eslintrc.json

```diff
@@ -1,158 +1,155 @@
 {
-  "extends": [
```

Comment:
please undo the changes to `webapp/.eslintrc.json` and update the PR without formatting changes